.golangci.yml

```diff
@@ -56,6 +56,7 @@ linters-settings:
     # List of functions to exclude from checking, where each entry is a single function to exclude.
     # See https://github.com/kisielk/errcheck#excluding-functions for details.
     exclude-functions:
+      - Test
```

Comment:
Is it a regex? I was checking the linked doc and it did not mention it's a regex or pattern matching. I'm just wondering if this line has indeed the desired behaviour.

Reply:
Sorry, a left-over from testing.